### PR TITLE
add to_imperative method on each collection

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: dask/array/core.py [hunk 1]
```diff
@@ -1348,6 +1348,14 @@ def view(self, dtype, order='C'):
         out._chunks = chunks
         return out
 
+    def to_imperative(self):
+        """ Convert Array into dask Values
+
+        Returns a nested list of values, one value per chunk.
+        """
+        from ..imperative import Value
+        return deepmap(lambda k: Value(k, [self.dask]), self._keys())
```
Comment (shoyer) [inline dask/array/core.py]: maybe an array with `dype=object` would be more useful?

Comment (mrocklin) [inline dask/array/core.py]: Fixed

**Comment by mrocklin** [comment]
Merging soon if no comments.

**Comment by jcrist** [comment]
Looks good to me. Array of `Value`'s is a good idea. +1.

**Pull request merged**